mlir/lib/Pass/PassManagerOptions.cpp

```diff
@@ -131,8 +131,12 @@ void mlir::registerPassManagerCLOptions() {
 }
 
 LogicalResult mlir::applyPassManagerCLOptions(PassManager &pm) {
-  if (!options.isConstructed())
+  if (!options.isConstructed()) {
+    emitError(UnknownLoc::get(pm.getContext()))
+        << "could not apply pass manager command line options.\n"
+           "Missing 'registerPassManagerCLOptions' call?\n";
     return failure();
```

Comment:
Isn't the following the idiomatic way to write this?
```suggestion
    return emitError(UnknownLoc::get(pm.getContext()))
        << "could not apply pass manager command line options.\n"
           "Missing 'registerPassManagerCLOptions' call?\n";
```